sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -78,6 +78,8 @@ final class ManifestMetadataReader {
   static final String CLIENT_REPORTS_ENABLE = "io.sentry.send-client-reports";
   static final String COLLECT_ADDITIONAL_CONTEXT = "io.sentry.additional-context";
 
+  static final String PERFORM_ACTIVITY_FRAMES_TRACKING = "io.sentry.activity-frames-tracking";
```

Comment:
Not sure if the string should be suffixed with `".enable"`, like some of the other attributes

Reply:
The `.enable` is legacy, I'd rather not do it, just verbose.